roles/consul/tasks/set_acl_token.yml

```diff
@@ -1,11 +1,12 @@
-- name: "Set consul token in server.hcl"
-  replace:
-    path: "{{ consul_config }}server.hcl"
-    replace: "{{ agent_consul_token }}"
-    regexp: '{{ default_token }}'
+- name: "Copy acl.hcl configuration"
+  template:
+   src: templates/consul_acl.j2
+   dest: "{{ consul_config }}acl.hcl"
+   owner: "{{ consul_username }}"
+   force: no
   notify: Restart consul
 
```

Comment:
Why is this file needed? When the "Set agent token" task executes it creates a file in `/opt/consul` called `acl-tokens.json` wich contains the token passed to the  `consul acl set-agent-token agent` command. This file is the one used to obtain the token when needed.